src/components/grid-aware/Hero/Hero.module.css

```diff
@@ -0,0 +1,76 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding-bottom: 150px;
+  padding-top: 150px;
```

Comment:
If left and right padding are 0 by default, would it be better to short-hand `padding` property such as `padding: [top] [right-left] [bottom];` or `padding: 150px 0 150px;`?